client/src/components/search/Search.tsx

```diff
@@ -1,7 +1,6 @@
 import React, { useEffect, useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
-import Input from "../Input/Input";
-import Button from "../Button/Button";
+import Input from "../input/Input";
```

Comment:
This should be `import Input from "../Input/Input";`, not `import Input from "../input/Input";`